lib/queries/github_user_info.graphql

```diff
@@ -0,0 +1,6 @@
+query ($user: String!) {
```

Comment:
Remove space before `(`
(or add a space on the other ones, I'm not 100% sure which is preferred in GraphQL, but I think I've seen no space used more frequently)

```suggestion
query($user: String!) {
```